server/src/common/dto/get-applications.dto.ts

```diff
@@ -0,0 +1,7 @@
+export declare class GetApplicationsDto {
```

Comment:
Could you add validations? Also making the Pagination DTO separately and extend them here!